CompetitionTemplate/src/competition/subsystems/wrist/wrist_commands/MoveWristDownCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.wrist.wrist_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.wrist.WristSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class MoveWristDownCommand extends BaseCommand{
+    WristSubsystem wristSubsystem;
+    
+    @Inject
+    public MoveWristDownCommand (WristSubsystem wristSubsystem) {
+        this.wristSubsystem = wristSubsystem;
+        this.requires(this.wristSubsystem);
+    }
+
+    @Override
+    public void execute() {
+        
+    }
+
+    @Override
+    public void initialize() {
+        wristSubsystem.moveWristDown();
+    }
+    
```

Comment:
Hypothetically speaking, these commands could finish as soon as this runs so let's throw in an isFinished that returns true all the time